core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/event/span/export/optel/OpenTelemetryResourcesProvider.java

```diff
@@ -41,59 +46,64 @@
 public class OpenTelemetryResourcesProvider {
 
 
-  private static CapturingSpanExporterWrapper capturingSpanExporterWrapper;
+  private final static CapturingSpanExporterWrapper capturingSpanExporterWrapper =
+      new CapturingSpanExporterWrapper(new SpanExporter() {
+
+        @Override
+        public CompletableResultCode export(Collection<SpanData> collection) {
+          return new CompletableResultCode().succeed();
+        }
+
+        @Override
+        public CompletableResultCode flush() {
+          return new CompletableResultCode().succeed();
+        }
+
+        @Override
+        public CompletableResultCode shutdown() {
+          return new CompletableResultCode().succeed();
+        }
+      });
+
   private static final String OPENTELEMETRY_EXPORT_ENABLED_SYSPROP = "mule.openetelemetry.export.enabled";
   private static final String MULE_OPENTELEMETRY_ENDPOINT_SYSPROP = "mule.opentelemetry.endpoint";
+  // This is only defined in the semconv artifact which is in alpha state and is only needed for this.
+  // In order not to add another dependency we add it here.
+  // For the moment it is defined in the spec here:
+  // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#semantic-attributes-with-dedicated-environment-variable
+  public static final AttributeKey<String> SERVICE_NAME_KEY = stringKey("service.name");
```

Comment:
Let's add a TODO with a task for tracking this (maybe we never add the dependency, maybe not)